cmd/podman/rmi.go

```diff
@@ -53,12 +53,24 @@ func rmiCmd(c *cli.Context) error {
 
 	images := args[:]
 	var lastError error
+	var imagesToCheck []*image.Image
 	var imagesToDelete []*image.Image
 	if removeAll {
-		imagesToDelete, err = runtime.ImageRuntime().GetImages()
+		imagesToCheck, err = runtime.ImageRuntime().GetImages()
 		if err != nil {
 			return errors.Wrapf(err, "unable to query local images")
 		}
+		// Look up the image by it's repository rather than its ID.
+		// We need this later if we need to remove for Storage (Buildah).
```

Comment:
Why is this necessary? Can we not find the image by its ID?

Reply:
I did this for the case when a image was tagged.  If I do:
```
podman pull alpine
podman tag alpine tg
podman images
REPOSITORY                 TAG      IMAGE ID       CREATED        SIZE
tg                         latest   3fd9065eaf02   2 months ago   4.41MB
docker.io/library/alpine   latest   3fd9065eaf02   2 months ago   4.41MB
```
Both "images" had the same ID, the only way I could differentiate them was to go by their repo.